src/generator/AutoRest.AzureResourceSchema/CodeGeneratorArs.cs

```diff
@@ -29,12 +29,35 @@ public CodeGeneratorArs()
             {
                 StringWriter stringWriter = new StringWriter();
                 ResourceSchemaWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
-                await Write(stringWriter.ToString(), resourceProvider + ".json", true);
+                await Write(stringWriter.ToString(), resourceProvider + Path.DirectorySeparatorChar + resourceProvider + ".json", true);
 
                 stringWriter = new StringWriter();
-                ResourceMarkdownWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
-                await Write(stringWriter.ToString(), resourceProvider + ".md", false);
+                var md = ResourceMarkdownGenerator.Generate(resourceSchemas[resourceProvider]);
+
+                foreach (var m in md)
+                {
+                    var content = m.Content.Replace("\"boolean\"", "boolean");
```

Comment:
This is the way the doc team wanted this part of the template to look (it does make it a bit easier to read).